Automattic/wp-calypso
# Add anchor-fm to quicklinks

### Known issues

- **We're not sure we're going to add it**
- Possibly get an inverted logo from them, this comes from a modified
  version of the SVG in their presskit, and doesn't match mockups
- Text spans two lines in my browser
- Need a better destination URL

#### Changes proposed in this Pull Request

* Add anchor fm link to quick links section in calypso home

#### Testing instructions

* Ensure `anchor-fm` feature is on
* Visit calypso home
* Look for new quick link
![2020-11-18_11-40](https://user-images.githubusercontent.com/937354/99566728-e5a9b300-2992-11eb-8409-be80be4f8f87.png)
